etc/config/fortran.amazon.properties

```diff
@@ -62,12 +62,12 @@ group.ifort.intelAsm=-masm=intel
 group.ifort.groupName=IFORT x86-64
 group.ifort.isSemVer=true
 group.ifort.baseName=x86-64 ifort
-compiler.ifort16.exe=/opt/compiler-explorer/intel/xe_2016_update3/bin/ifort
-compiler.ifort16.semver=16.0.3
-compiler.ifort17.exe=/opt/compiler-explorer/intel/2017/bin/ifort
-compiler.ifort17.semver=17.0.0
-compiler.ifort18.exe=/opt/compiler-explorer/intel-2018.0.033/bin/ifort
-compiler.ifort18.semver=18.0.0
+#compiler.ifort16.exe=/opt/compiler-explorer/intel/xe_2016_update3/bin/ifort
+#compiler.ifort16.semver=16.0.3
+#compiler.ifort17.exe=/opt/compiler-explorer/intel/2017/bin/ifort
+#compiler.ifort17.semver=17.0.0
+#compiler.ifort18.exe=/opt/compiler-explorer/intel-2018.0.033/bin/ifort
+#compiler.ifort18.semver=18.0.0
```

Comment:
Got disabled in https://github.com/compiler-explorer/compiler-explorer/pull/1129#pullrequestreview-169847158 but never reenabled, even though we seem to have the executables available?

Reply:
Oh wow! Worth another look!